src/mlpack/methods/ann/rbm/rbm.hpp

```diff
@@ -86,11 +86,14 @@ class RBM
    * optimization. If this is not what you want, then you should access the
    * parameters vector directly with Parameters() and modify it as desired.
    *
+   * @tparam OptimizerType Type of optimizer to use to train the model.
+   * @tparam CallbackTypes Types of Callback functions.
    * @param optimizer Optimizer type.
+   * @param callbacks Callback Functions.
```

Comment:
In another PR, @rcurtin suggested to add a reference to the ensmallen documentation: #2074 (comment). I think we should do the same here as well.